include/range/v3/numeric/partial_sum.hpp

```diff
@@ -50,19 +48,21 @@ namespace ranges
             tagged_pair<tag::in(I), tag::out(O)>
             operator()(I begin, S end, O result, S2 end_result, BOp bop = BOp{}, P proj = P{}) const
```

Comment:
Pre-existing: `S` and `S2` is weird. We should rename `S` to `S1` while we're in here.